lib/local/platform/windows.js

```diff
@@ -36,6 +44,15 @@ module.exports = {
     pathQuery: 'dir /s /b iexplore.exe',
     cwd: cwd
   },
+  edge: {
+    defaultLocation: path.join(microsoftEdgeDirectory, 'MicrosoftEdge.exe'),
```

Comment:
I guess this should be `defaultLocation: microsoftEdgeDirectory ? path.join(microsoftEdgeDirectory, 'MicrosoftEdge.exe') : null` to avoid putting in a bad path on older versions of Windows.